# Allow to filter and select pose(s) by information and/or parameters

# Select the(all) start pose(s)
bool polygon_start

# Select the(all) end pose(s)
bool polygon_end

# Select the(all) polygon start and end pose(s)
bool both_start_end

# Select the last pose of the layer
bool last_pose_in_layer
